test/e2e/common/volumes.go

```diff
@@ -91,7 +91,7 @@ var _ = Describe("[sig-storage] GCP Volumes", func() {
 			}
 
 			// Must match content of test/images/volumes-tester/nfs/index.html
-			framework.TestVolumeClient(c, config, nil, tests)
```

Comment:
nit, add comment a la `framework.TestVolumeClient(c, config, nil, "" /* fsType */, tests)` for unnamed variables

Reply:
Thank you for your comment. I will fix all of them as suggested.